### PR TITLE
[#177017837] Added new api for retrieving eyca card

### PR DESCRIPTION
**Pull request by gquadrati**
#### List of Changes
- Add `getEycaStatus` API
- Add tests

#### Motivation and Context
While a citizen requests for a CGN and he's between 18 up to 30 years old, he wants also activate an EYCA card ( CGN and EYCA are co-badged ). We need to provide an API that could return detailed information about an EYCA Card.

#### How Has This Been Tested?
It has been tested by performing unit tests .

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: GetEycaStatus/function.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "route": "api/v1/cgn/status/{fiscalcode}",
```
Comment (balanza) [inline GetEycaStatus/function.json:8]: Isn't it "api/v1/cgn/eyca/status/{fiscalcode}"?

Comment (gquadrati) [reply GetEycaStatus/function.json:8]: Sure it is

Comment (gquadrati) [reply GetEycaStatus/function.json]: Done in [63360ab](https://github.com/pagopa/io-functions-cgn/pull/25/commits/63360abb4958e58b6b0c5a10fb9d0c8b51b91d64)

**Review by AleDore — APPROVED** [review]
LGTM

**Pull request merged**